### ISSUE TITLE
[wrgd] switch to add dataset to column titles

### ISSUE THREAD

**Issue by JakubMatejka** [keboola/kbc-ui#485]
There should be a switch on whole writer's configuration level (i.e. somewhere on page with tables list) which automatically adds dataset to column titles (e.g. column Name in dataset Products will have title "Name (Products)". It is attribute `addDatasetTitleToColumns` in writer's bucket and can be set using API call http://docs.keboolagooddatawriterv2.apiary.io/#reference/writers/writers/update-writer's-configuration and read using API call http://docs.keboolagooddatawriterv2.apiary.io/#reference/writers/writers/get-writer

This is switch for one-time use so I'm not sure if right column on page with tables list is the best place. There could be some dedicated page (or popup) with settings but this should be probably well considered with other Writers in mind (maybe it's task for UX guy).

**Comment by kacurez** [comment]
Neviem si to z hladiska ux moc dobre predstavit:
Da sa zistit ci ten prepinac `addDatasetTitleToColumns ` uz bol nastaveny?
Nebolo by lepsie to nastavovat len pri zakladani writeru?

**Comment by JakubMatejka** [comment]
Já taky ne. ;o) Vlastně možná i jo, asi by se to v průběhu ani nemělo moc měnit. @padak ?

**Comment by padak** [comment]
No, v nějakým "settings" writeru bych viděl přepínač on/off, kterej by to nastavil. Z pohledu usera by to bylo třeba tak že ten title je má preffix, kterej se zapíná tímhle přepínačem a pak má value, který nastavuje user. Není to řešení?

**Comment by padak** [comment]
Navrhuju to dát k šípku, bo nevíme co s tím a vydrželo to víc jak rok. Co vy na to?
